pyscript.core/tests/integration/test_py_config.py

```diff
@@ -30,7 +31,7 @@ def test_py_config_inline_pyscript(self):
         )
         assert self.console.log.lines[-1] == "config name: foobar"
 
-    @pytest.mark.skip("ERROR_SCRIPT: works with <py-script> not with <script>")
+    @pytest.mark.skip("NEXT: works with <py-script> not with <script>")
```

Comment:
this is misleading ... you are accessing `document.currentScript._pyodide.promise` to grab the `config` and we landed already an explicit *export* of the config we pass along ... this needs a rewrite, happy to help if needed, you need to `import { config as pyConfig } from 'core.js' and then `globalThis.pyConfig = pyConfig` and change any code that use internals wrongly to reach the *config*.